modules/ROOT/pages/_partials/data-sync/ref_terminology.adoc

```diff
@@ -1,14 +1,14 @@
 //':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
-[id='ref_terminology_{context}']
+[id='ref-data-sync-terminology-{context}']
+= Data Sync terminology
 
-= {sync-service} Terminology
+This section describes terminology that is associated with Data Sync.
```

Comment:
needed?

Reply:
Yes, "A short introductory paragraph is required for the reference module. It will provide an overview of the module." (https://github.com/redhat-documentation/modular-docs/blob/master/modular-docs-manual/files/TEMPLATE_REFERENCE_reference-material.adoc)